src/test/scala/minitime/BaseSpec.scala

```diff
@@ -0,0 +1,5 @@
+package minitime
+
+import org.scalatest.{Matchers, WordSpec}
+
+abstract class BaseSpec extends WordSpec with Matchers
```

Comment:
Can we change to use FlatSpec since ScalaTest recommends it as the default choice?

Reply:
Sure.